src/style.css

```
@@ -0,0 +1,121 @@
+:root {
+  --text-color: #3d4e61;
+  --border-color: #e1e7ed;
+  --box-shadow: rgba(61,78,97,0.06);
+  --box-shadow-focus-hover: rgba(61,77,97,0.13);
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(fonts/Avenir.ttc);
+  font-style: normal;
+  font-weight: 400;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-style: normal;
+  font-weight: 800;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+.big {
+  display: block;
+  position: relative;
+  margin: 20px 0;
+}
+
+html .big-input {
+  border: 1px solid var(--border-color);
+  border-radius: 4px;
+  box-shadow: 0 1px 8px 0 var(--box-shadow);
+  padding: 24px 25px 24px 62px;
+  width: 100%;
+  font-family: "Avenir", sans-serif;
```

set font-family to the `html` tag selector and let inputs inherit it
```suggestion
  font-family: inherit;
```